### ISSUE TITLE
ErrorException in EntityCollection.php line 405

### ISSUE THREAD

**Issue by Nyorai** [analogueorm/analogue#135]
I get the following error when using 5.3-dev with Laravel 5.3

```
ErrorException in EntityCollection.php line 405: Declaration of Analogue\ORM\EntityCollection::unique($key = NULL) should be compatible with Illuminate\Support\Collection::unique($key = NULL, $strict = false)
```

Changing the declaration of unique to:

```
public function unique($key = null, $strict = false)
```

Fixes this for me, but not sure if it would cause any other issues.

**Comment by RemiCollin** [comment]
I just merged a PR which fixes this. Can you confirm this works ?

**Comment by joaoeduardo** [comment]
Have been fixed? I got same error.

**Comment by RemiCollin** [comment]
Yes, it's fixed in the 5.3-dev branch.

**Comment by joaoeduardo** [comment]
When will be released the next stable release?

**Comment by RemiCollin** [comment]
Should be around the end of the month

**Comment by RemiCollin** [comment]
Fixed in 5.3
